base.scss

```diff
@@ -0,0 +1,68 @@
+h1 {
+    font-family: Lucida Sans Unicode;
+    border-bottom: 1px solid silver;
+}
+
+#page-wrapper {
+    width: 500px;
+    position: absolute;
+    top: 40px;
+    left: 50%;
+    margin-left: -250px;
+    padding: 5px;
+}
+
+#heading {
+    font-size: 24px;
+    text-align: center
+}
+
+#errors {
+    border: 1px solid #777777;
+    padding: 3px;
+    display: none;
+}
+
+.row {
+    margin: 15px 0;
+}
+
+.row label {
+    display: inline-block;
+    float: left;
```

Comment:
As element is floated, display: inline-block is ignored, display: block is used instead